src/autogluon/bench/benchmark.py

```diff
@@ -34,3 +37,15 @@ def upload_metrics(self, s3_bucket: str, s3_dir: str):
                 s3.upload_file(local_path, s3_bucket, s3_path)
 
         logging.info("Metrics under %s has been saved to %s/%s.", self.metrics_dir, s3_bucket, s3_dir)
+
+    def cleanup_metrics(self):
+        shutil.rmtree(self.benchmark_dir)
+        if self.benchmark_dir_s3 is not None:
+            import boto3
+
+            s3 = boto3.resource("s3")
+            bucket_name = self.benchmark_dir_s3.split("//")[-1].split("/")[0]
+            benchmark_dir = self.benchmark_dir_s3.split(bucket_name)[-1].lstrip("/")
+            bucket = s3.Bucket(bucket_name)
+            bucket.objects.filter(Prefix=benchmark_dir).delete()
+            s3.Object(bucket_name, self.benchmark_dir_s3).delete()
```

Comment:
Is there any risk in these command that someone chooses a poorly named bucket/benchmark_dir combo and we accidentally delete someone else's important S3 data? Maybe phrase another way, are we guaranteed that the benchmark_dir is something we created and are allowed to delete?

Reply:
This is a very good point. The `cleanup_metrics` is an instance method that uses `self.benchmark_dir` which is created with timestamp by the initializer, for `self.benchmark_dir_s3`, it is currently defined by `f'{module_name}/{benchmark.benchmark_name}'` which is a path containing `self.benchmark_dir`.